implementations/rust/ockam/ockam_app_lib/src/invitations/commands.rs

```diff
@@ -196,93 +199,151 @@ impl AppState {
             invitations
         };
 
-        self.invitations().write().await.replace_by(invitations);
-        self.publish_state().await;
+        let changed = {
+            let invitations_arc = self.invitations();
+            let mut guard = invitations_arc.write().await;
+            guard.replace_by(invitations)
+        };
+
+        if changed {
+            self.publish_state().await;
+            self.schedule_inlets_refresh_now();
+        }
+
         Ok(())
     }
 
     pub(crate) async fn refresh_inlets(&self) -> crate::Result<()> {
-        debug!("Refreshing inlets");
+        info!("Refreshing inlets");
```

Comment:
We have an `info` log at the end of the function. Wouldn't it be too spammy to have both logs in the info level?

Reply:
i think having one log entry every 10 seconds is acceptable